### PR TITLE
Fix duplicate names in generated reactors

### LINKED ISSUE #34: All functions of a stub created via `atomkraft reactor` have the same name
... as a result, the reactor is non-functional. This needs to be fixed.

### LINKED ISSUE #44: `atomkraft reactor` generates methods with same names
`atomkraft reactor --actions "fizz,fuzz,fizzfuzz"  --variables "x,y"`generates the following code.

`act_step` should be different for each method.

```
import time
import pytest
from cosmos_net.pytest import Testnet
from modelator.pytest.decorators import step

    
keypath = 'action'



@pytest.fixture
def state():
    return {}


@step('fizz')
def act_step(testnet, state, x, y):
    print("Step: fizz")


@step('fuzz')
def act_step(testnet, state, x, y):
    print("Step: fuzz")


@step('fizzfuzz')
def act_step(testnet, state, x, y):
    print("Step: fizzfuzz")
```

### PR DESCRIPTION
**Pull request by andrey-kuprianov**
Closes #34 
Closes #44

### CODE REVIEW

**Review by hvanz — APPROVED** [review]
LGTM

File: atomkraft/reactor/reactor.py [hunk 1]
```diff
@@ -108,8 +108,8 @@ def _keypath_stub(keypath):
 
 def _action_stub(action_name: str, variables: List[str]):
     stub = f"""
-@step({repr(action_name)})
-def act_step(testnet, state, {", ".join(variables)}):
+@step({action_name})
+def {action_name}(testnet, state, {", ".join(variables)}):
```
Comment (rnbguy) [inline atomkraft/reactor/reactor.py]: `{action_name}` should be sanitized.

Something like [case-converter](https://pypi.org/project/case-converter) should be enough.
```
from caseconverter import snakecase

def {snakecase(action_name)}(testnet, state, {", ".join(variables)}):
```

Comment (andrey-kuprianov) [reply atomkraft/reactor/reactor.py]: Good catch, @rnbguy! Done. Could you please review the PR, and merge it if everything else is fine?

**Review by rnbguy — COMMENTED** [review]
LGTM :+1:

**Pull request merged**